test/Dapr.E2E.Test/DaprTestApp.cs

```diff
@@ -19,8 +21,8 @@ public class DaprTestApp
         static string daprBinaryName = "dapr";
         private string appId;
         private bool useAppPort;
-        const string outputToMatchOnStart = "dapr initialized. Status: Running.";
-        const string outputToMatchOnStop = "app stopped successfully";
+        private readonly string[] outputToMatchOnStart = new string[]{ "You're up and running", };
```

Comment:
The issue with this string was that this output appears only when the sidecar starts...the app has not necessarily started and been discovered. The service invocation calls might fail at that point. "dapr initialized. Status: Running." is shown after dapr and the app are fully up

Reply:
OK cool. Will revert this. Ultimately I think it's the health check that made a difference.